### DISCUSSION TITLE
Show love for the drop-down

### DISCUSSION THREAD

**Discussion by IdaEdz** [ONLYOFFICE/DocumentServer#2527] [Ideas]
Hey!

Our users are moving some documents from G docs to OnlyOffice.
And in sheets they are used to that arrow for selecting values in a drop down list with the data validation tool.

In OnlyOffice right now (version 7.4) the arrow is only shown when hovering above with the mouse (as is the title and description, if set).
I think the dropdown arrow should be always on and inside the cell, not outside the cell.

**Comment by Rita-Bubnova** [comment]
Sorry for the long wait.
DocumentServer v9.3.1 is released. Is this problem relevant?
